<ui version="4.0" >
 <author></author>
 <comment></comment>
 <exportmacro></exportmacro>
 <class>BrightnessContrastWidget</class>
 <widget class="QWidget" name="BrightnessContrastWidget" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>695</width>
    <height>69</height>
   </rect>
  </property>
  <property name="windowTitle" >
   <string>Form</string>
  </property>
  <layout class="QGridLayout" >
   <property name="margin" >
    <number>8</number>
   </property>
   <property name="spacing" >
    <number>6</number>
   </property>
   <item row="0" column="2" >
    <widget class="QCheckBox" name="chkPreview" >
     <property name="text" >
      <string>Preview</string>
     </property>
    </widget>
   </item>
   <item row="1" column="1" >
    <layout class="QHBoxLayout" >
     <property name="margin" >
      <number>0</number>
     </property>
     <property name="spacing" >
      <number>6</number>
     </property>
     <item>
      <widget class="QSlider" name="sldContrast" >
       <property name="minimum" >
        <number>-100</number>
       </property>
       <property name="maximum" >
        <number>100</number>
       </property>
       <property name="orientation" >
        <enum>Qt::Horizontal</enum>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QSpinBox" name="spnContrast" >
       <property name="maximum" >
        <number>100</number>
       </property>
       <property name="minimum" >
        <number>-100</number>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="1" column="0" >
    <layout class="QHBoxLayout" >
     <property name="margin" >
      <number>0</number>
     </property>
     <property name="spacing" >
      <number>6</number>
     </property>
     <item>
      <widget class="QSlider" name="sldBrightness" >
       <property name="minimum" >
        <number>-100</number>
       </property>
       <property name="maximum" >
        <number>100</number>
       </property>
       <property name="orientation" >
        <enum>Qt::Horizontal</enum>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QSpinBox" name="spnBrightness" >
       <property name="maximum" >
        <number>100</number>
       </property>
       <property name="minimum" >
        <number>-100</number>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="0" column="1" >
    <widget class="QLabel" name="label_2" >
     <property name="text" >
      <string>&lt;html>&lt;head>&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; />&lt;/head>&lt;body style=&quot; white-space: pre-wrap; font-family:Sans Serif; font-weight:400; font-style:normal; text-decoration:none;&quot;>&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;>Contrast&lt;/p>&lt;/body>&lt;/html></string>
     </property>
    </widget>
   </item>
   <item row="1" column="2" >
    <layout class="QHBoxLayout" >
     <property name="margin" >
      <number>0</number>
     </property>
     <property name="spacing" >
      <number>6</number>
     </property>
     <item>
      <widget class="QPushButton" name="btnOk" >
       <property name="text" >
        <string>OK</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btnCancel" >
       <property name="text" >
        <string>Cancel</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="0" column="0" >
    <widget class="QLabel" name="label" >
     <property name="text" >
      <string>&lt;html>&lt;head>&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; />&lt;/head>&lt;body style=&quot; white-space: pre-wrap; font-family:Sans Serif; font-weight:400; font-style:normal; text-decoration:none;&quot;>&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;>Brightness&lt;/p>&lt;/body>&lt;/html></string>
     </property>
    </widget>
   </item>
  </layout>
  <widget class="QWidget" name="" >
   <property name="geometry" >
    <rect>
     <x>268</x>
     <y>9</y>
     <width>148</width>
     <height>50</height>
    </rect>
   </property>
  </widget>
  <widget class="QWidget" name="" >
   <property name="geometry" >
    <rect>
     <x>526</x>
     <y>9</y>
     <width>160</width>
     <height>57</height>
    </rect>
   </property>
  </widget>
  <widget class="QWidget" name="" >
   <property name="geometry" >
    <rect>
     <x>9</x>
     <y>9</y>
     <width>253</width>
     <height>58</height>
    </rect>
   </property>
  </widget>
 </widget>
 <pixmapfunction></pixmapfunction>
 <resources/>
 <connections>
  <connection>
   <sender>sldBrightness</sender>
   <signal>valueChanged(int)</signal>
   <receiver>spnBrightness</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>80</x>
     <y>53</y>
    </hint>
    <hint type="destinationlabel" >
     <x>218</x>
     <y>58</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>sldContrast</sender>
   <signal>valueChanged(int)</signal>
   <receiver>spnContrast</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>414</x>
     <y>57</y>
    </hint>
    <hint type="destinationlabel" >
     <x>477</x>
     <y>52</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>spnBrightness</sender>
   <signal>valueChanged(int)</signal>
   <receiver>sldBrightness</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>252</x>
     <y>43</y>
    </hint>
    <hint type="destinationlabel" >
     <x>45</x>
     <y>59</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>spnContrast</sender>
   <signal>valueChanged(int)</signal>
   <receiver>sldContrast</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>484</x>
     <y>42</y>
    </hint>
    <hint type="destinationlabel" >
     <x>329</x>
     <y>57</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
